CMake/CreateArmaConfigInfo.cmake

```diff
@@ -10,36 +10,6 @@ else()
   set(OLD_FILE_CONTENTS "")
 endif()
 
-# If we are using Armadillo 5+, ARMA_64BIT_WORD is implicitly enabled.
-set(ARMA_HAS_64BIT_WORD 0) # This may be unnecessary.
-if(NOT (${ARMADILLO_VERSION_MAJOR} LESS 5))
-  # ARMA_64BIT_WORD is only set if we are on a 64-bit system.
-  if (CMAKE_SIZEOF_VOID_P EQUAL 8)
-    set(ARMA_HAS_64BIT_WORD 1)
-  else ()
-    set(ARMA_HAS_64BIT_WORD 0)
-  endif ()
```

Comment:
Don't we need to have the `CMAKE_SIZEOF_VOID_P` check in this file too, so that `ARMA_HAS_64BIT_WORD` is defined for the check below?